### PR TITLE
feat: add explorer url config to networks

### PR DESCRIPTION
**Pull request by luizstacio**
- Add explorer url configuration for networks, Users are able to change this on `edit` network screen.
- Add faucet url on hardcoded list to show faucet when asset list is empty.
- Add bridge url on hardcoded list to show bridge when asset list is empty.

### CODE REVIEW

File: packages/app/src/config.ts [hunk 1]
```diff
@@ -38,5 +38,5 @@ export const IS_CRX_POPUP =
 /** Time in minutes before Wallet auto locks */
 export const AUTO_LOCK_IN_MINUTES = VITE_AUTO_LOCK_IN_MINUTES;
 export const MIN_NODE_VERSION = '0.33.0';
-export const TESTNET_URL = 'https://testnet.fuel.network/v1/graphql';
-export const DEVNET_URL = 'https://devnet.fuel.network/v1/graphql';
+export const IS_RELEASE =
+  !IS_DEVELOPMENT && process.env.VITE_CRX_RELEASE === 'true';
```
Comment (arthurgeron) [inline packages/app/src/config.ts:42]:
```suggestion
  !IS_DEVELOPMENT && !IS_TEST && process.env.VITE_CRX_RELEASE === 'true';
```

File: packages/app/src/networks.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+];
+
+if (
+  IS_DEVELOPMENT &&
+  !DEFAULT_NETWORKS.find((n) => n.url === VITE_FUEL_PROVIDER_URL)
+) {
+  DEFAULT_NETWORKS.push({
+    name: 'Local',
+    url: VITE_FUEL_PROVIDER_URL,
+    chainId: 9999,
```
Comment (arthurgeron) [inline packages/app/src/networks.ts:46]: Shouldn't it be equal to testnet's?

File: packages/app/src/networks.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    url: 'https://devnet.fuel.network/v1/graphql',
+    chainId: CHAIN_IDS.fuel.devnet,
+    explorerUrl: 'https://app-devnet.fuel.network',
+    faucetUrl: 'https://faucet-devnet.fuel.network/',
+    isSelected: false,
+  },
+];
+
+if (
+  IS_DEVELOPMENT &&
```
Comment (arthurgeron) [inline packages/app/src/networks.ts:40]:
```suggestion
  (IS_DEVELOPMENT || IS_TEST) &&
```

File: packages/app/src/systems/FundWallet/hooks/useFundWallet.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import { openTab } from '~/systems/CRX/utils';
+import { useNetworks } from '~/systems/Network';
+
+export function useFundWallet() {
+  const { selectedNetwork } = useNetworks();
+  const { account } = useAccounts();
+
+  const { faucetUrl, bridgeUrl } = useMemo(() => {
+    const network = DEFAULT_NETWORKS.find(
+      (n) => n.url === selectedNetwork?.url
```
Comment (arthurgeron) [inline packages/app/src/systems/FundWallet/hooks/useFundWallet.tsx:15]: Shouldn't the selected network obj come ready to use from useNetworks/NetworkMachine?

**Review by arthurgeron — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/FundWallet/hooks/useFundWallet.tsx (hunk 4 above)
Comment (luizstacio) [reply packages/app/src/systems/FundWallet/hooks/useFundWallet.tsx:15]: It dosen't have the bridge or facuet url, and I don't think we should add to the database.

**Review by LuizAsFight — APPROVED** [review]

**Review by arthurgeron — APPROVED** [review]

**Pull request merged**